libs/core/langchain_core/tracers/langchain.py

```diff
@@ -69,11 +71,16 @@ def _get_executor() -> ThreadPoolExecutor:
 
 
 def _run_to_dict(run: Run) -> dict:
-    return {
-        **run.dict(exclude={"child_runs", "inputs", "outputs"}),
-        "inputs": run.inputs.copy() if run.inputs is not None else None,
-        "outputs": run.outputs.copy() if run.outputs is not None else None,
-    }
+    # TODO: Update once langsmith moves to Pydantic V2 and we can swap run.dict for
+    # run.model_dump
+    with warnings.catch_warnings():
```

Comment:
should double check that this is actually needed, was messing around with a local version of langsmith that i upgraded to v2